scripts/CMakeLists.txt

```diff
@@ -251,6 +251,7 @@ SET(mariadb-hotcopy_COMPONENT COMPONENT Client)
 SET(mariadb-convert-table-format_COMPONENT COMPONENT Client)
 SET(mariadb-setpermission_COMPONENT COMPONENT Client)
 SET(mariadb-secure-installation_COMPONENT COMPONENT Client)
+SET(mariadb-dumpslow_COMPONENT COMPONENT Client)
```

Comment:
This is also ending with no binary in installation directory. I will append a patch.
```
$ find /tmp/10.6_without_server/|grep dumpslow
/tmp/10.6_without_server/man/man1/mariadb-dumpslow.1
/tmp/10.6_without_server/man/man1/mysqldumpslow.1
```